package.json

```diff
@@ -49,10 +49,11 @@
     "@aws-sdk/types": "^3.0.0"
   },
   "devDependencies": {
-    "@aws-sdk/client-dynamodb": "3.18.0",
-    "@aws-sdk/client-sns": "3.18.0",
-    "@aws-sdk/client-sqs": "3.18.0",
-    "@aws-sdk/lib-dynamodb": "3.18.0",
+    "@aws-sdk/types": "^3.18.0",
+    "@aws-sdk/client-dynamodb": "^3.18.0",
+    "@aws-sdk/client-sns": "^3.18.0",
+    "@aws-sdk/client-sqs": "^3.18.0",
+    "@aws-sdk/lib-dynamodb": "^3.18.0",
```

Comment:
Keep the latest versions updated for local tests. We may as well just pull in the latest for tests to run against those.

Reply:
Keep in mind that the `^` doesn't mean the "latest" will be installed when you run `npm / yarn install`. If you install dependencies today and a new version will be released tomorrow, running `yarn install` again **will not** install the fresh versions, since the matching ones are already installed.

This is [one of the reasons](https://betterdev.blog/pin-exact-dependency-versions/) why I prefer to specify exact dependency versions (in open source only dev dependencies ofc).

But let's keep it that way, I may just change it later.